nodejs/Users.js

```diff
@@ -0,0 +1,43 @@
+
+// var bodyParser = require('body-parser');
+// var express = require('express');
+// var app = express();
+
+const axios = require('axios').default;
+const pilotUtils = require('./pilotUtils.js')
+const { exit } = require('process')
+
+const args = process.argv.slice(2);
+const uuid = '156b9cujydh6dq9s44dubnsg7r'; //args[0];

```

Comment:
do we want these hardcoded in here? or passed in as args?  this looks like it was just in place for testing :-)

Reply:
I see. I forgot that I had played with that file as well. Good catch!